### ISSUE TITLE
"State 'archive.extracted' is unavailable" on 2014.7.2 on Windows

### ISSUE THREAD

**Issue by dogrover** [saltstack/salt#22269]
I've seen several issues about the state `archive.extracted` not working in 2014.7.1 and earlier:

- #13077 archive state not working
- #20852 Not able to extract archive in 2014.7.1
- #21341 2014.7.1 regression. archive.extracted does not work

Those issues all seem to have been resolved, but I'm seeing something similar on 2014.7.2, on Windows:

```
C:\>salt-call --local --log-level=debug state.single archive.extracted name=C:\Temp source=C:\Temp\Archive.zip
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Configuration file path: c:\salt\conf\minion
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[DEBUG   ] Mako not available
[DEBUG   ] Reading configuration from c:\salt\conf\minion
[INFO    ] Loading fresh modules for state activity
[DEBUG   ] Failed to import returner :
Traceback (most recent call last):
  File "salt/loader.py", line 624, in gen_module
ImportError: No module named
local:
    Data failed to compile:
----------
    Specified state 'archive.extracted' is unavailable.

C:\>salt-minion --versions-report
           Salt: 2014.7.2
         Python: 2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]
         Jinja2: 2.7.1
       M2Crypto: 0.21.1
 msgpack-python: 0.4.2
   msgpack-pure: Not Installed
       pycrypto: 2.6
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.1.1
           RAET: Not Installed
            ZMQ: 4.0.4
           Mako: Not Installed
```

Perhaps unrelated: is that "Failed to import returner" message expected?

**Comment by rallytime** [comment]
@dogrover Do you know if this function was working in an earlier release on windows? I have a suspicion that the root cause here is perhaps the fact that the module function `archive.unzip` is not available in windows and perhaps archive.extracted also wasn't written with windows in mind. See issue #12992.

If it was working in windows, then let's get this bug fixed. If it's the case that this hasn't ever worked for windows, I'd be happy to approve this as a feature request.

**Comment by dogrover** [comment]
I'm not sure if it was ever available on Windows, but after looking at #12992 and the code, I'm willing to close this a duplicate of that one.

Thanks.
